src/Login.js

```diff
@@ -239,3 +183,32 @@ export default class Login {
       return client.getSsoLoginUrl(url.format(parsedUrl), loginType);
     }
 }
+
+
+/**
+ * Send a login request to the given server, and format the response
+ * as a MatrixClientCreds
+ *
+ * @param {string} hs   the base url of the Homeserver used to log in.
+ * @param {string} is   the default identity server
```

Comment:
I assume URL?

It might also be good to have these be named `hsUrl` and `isUrl` for self-documentation reasons.